metabase/metabase
# Cleanup new homepage

Epic: https://github.com/metabase/metabase/issues/21322

Changes:
- Adds schema selector in case there are multiple schemas in a database
- Adds screen size breakpoints
- Adds unit tests
- Adds e2e tests

How to test (1 option):
- Run a fresh Metabase instance
- During the setup, add a database with multiple schemas
- Make sure that after the setup you can switch between x-rays for different schemas with `Select`
- Make that margins and paddings become smaller on mobile screens

How to test (2 option):
- Run a fresh Metabase instance
- During the setup, do not add a database
- Change https://github.com/metabase/metabase/blob/89d2e878cbe95440a2b854216707bceb9373e73e/frontend/src/metabase/home/homepage/components/HomeXraySection/HomeXraySection.tsx#L28 to `const isSample = false;`
- Change `const canSelectSchema = schemas.length > 1;` to `const canSelectSchema = true`;
- Make sure that after the setup you can switch between x-rays for different schemas with `Select`
- Make that margins and paddings become smaller on mobile screens

<img width="1387" alt="Screenshot 2022-04-06 at 01 10 29" src="https://user-images.githubusercontent.com/8542534/161850352-677677f8-d1e8-4c11-a4d9-c1aeb2f75688.png">

<img width="505" alt="Screenshot 2022-04-05 at 00 24 17" src="https://user-images.githubusercontent.com/8542534/161625933-8b20d483-a6f4-4894-83d1-196030dead3d.png">
